lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -2739,6 +2751,123 @@ $attribute_list::5
     <xs:attributeGroup ref="AssertAttributePath"/>
     <xs:attributeGroup ref="AssertAttributeNegate"/>
   </xs:complexType>
+  <xs:complexType name="AssertImageHasMetadata">
+    <xs:annotation>
+      <xs:documentation xml:lang="en"><![CDATA[
+Asserts the image has the specified metadata.
+
+```xml
+<image_has_metadata width="512" height="512" channels="3" />
```

Comment:
Maybe split into `<has_image_width>`, `<has_image_height>`, `<has_image_channels>` and add `min`, `max` and `delta` attributes?

Reply:
Done this in 00485ccc5294815c06e1d4d51d92d487f4133c65.